packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -294,9 +319,238 @@ const MultiCodeLayout = ({
   );
 };
 
+/**
+ * Generic styled-component Image utilities
+ */
+const Img = styled.img`
+  min-width: 100%;
+  min-height: 100%;
+  max-width: 100%;
+  object-fit: cover;
```

Comment:
I see your point and since this is the vertical image layout, it could be argued that adding an image in landscape orientation wouldn't be supported - unless the user wants to override props. It's not like there's a spec for this feature, so I am happy to go with whatever the consensus is. 

@gksander, would you care to weigh in on the level of "cropping" behavior we want to support?